syntax = "proto3";

package fixbug;

//定义由 protoc 生成 C++ Service 类
//protoc 生成 Service 相关的虚基类
option cc_generic_services = true;

message ResultCode
{
    int32 errcode = 1; 
    bytes errmsg = 2;
}

message LoginRequest
{
    bytes name = 1;
    bytes pwd = 2;
}

message LoginResponse
{
    ResultCode result = 1;
    bool success = 2;
}

// 定义服务接口
service UserServiceRpc
{
    rpc Login(LoginRequest) returns(LoginResponse);
}